### ISSUE TITLE
generating array of enums

### ISSUE THREAD

**Issue by tkoomzaaskz** [pateketrueke/json-schema-faker#29]
I'm trying to provide an example faker schema that would generate *an array of enumerations*. That is, I specify possible values of the enum and give minimum number of items in the array. I've tried:

```json
{
  "type": "array",
  "minItems": "15",
  "items": [
    {
      "enum": [ "red", "green", "blue" ]
    }
  ]
}
```

but it doesn't generate what I expect. It creates things like the following:

```
[
  "green"
]
```
or
```
[
  "red"
]
```
or
```
[
  "green"
]
```
etc. And I expect:

```
[
  "green",
  "red",
  "green",
  "blue",
  "red",
  "green",
  "blue",
  "red",
  ...
]
```

Does json-schema-faker not support that feature or am I doing something wrong?

**Comment by pateketrueke** [comment]
This works:

```json
{
  "type": "array",
  "items": {
    "enum": ["red", "green", "blue"]
  }
}
```

You are using the array-notation around the sub-schema with the enum, it means you are specifying exactly how many values will be generated: one item in the array.

**Comment by tkoomzaaskz** [comment]
Thank you for the response. Your solution works and returns random number of elements. But how can I specify *minimum* number of elements to it?

**Comment by pateketrueke** [comment]
Of course with `minItems` but since there is a bug with #30 you need to specify `maxItems` also as workaround.

**Comment by pateketrueke** [comment]
Fixed on `0.1.8` :beers:
